fio/lib/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "fio-wallet",
+  "version": "1.0.6",
+  "description": "",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "/dist"
+  ],
+  "scripts": {
```

Comment:
Let's use npm instead of yarn for this package since that's used more widely.

Reply:
Done